Dockerfile.cuda

```diff
@@ -0,0 +1,22 @@
+FROM nvidia/cuda:12.2.0-devel-ubuntu22.04
```

Comment:
There's a slight desync between the cuda image and the non-cuda image. nvidia/cuda does not have an alpine base. For consistency, the non-cuda could use ubuntu:22.04 instead of alpine